packages/edit-post/src/index.js

```diff
@@ -77,6 +78,7 @@ export function initializeEditor(
 	// Check if the block list view should be open by default.
 	// If `distractionFree` mode is enabled, the block list view should not be open.
```

Comment:
Should this comment and the one in the site editor also be updated to reflect the change in logic?